shared/memorypool/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["memorypool.go"],
+    importpath = "github.com/prysmaticlabs/prysm/shared/memorypool",
+    visibility = ["//visibility:public"],
```

Comment:
visibility

Reply:
anything wrong with it ? it makes sense to make this public as it is in shared

Reply:
ah sorry, thought it was in beacon-chain